gradle/dependencies.gradle

```diff
@@ -17,8 +17,8 @@ println("kotlin version: $kotlinVersion")
 
 def versions = [
         dagger                         : '2.37',
-        intellij                       : '2020.1.4',
-        intellij_kotlin                : '1.3.72-release-IJ2020.1-1',
+        intellij                       : '203.7717.56',
+        intellij_kotlin                : '203-1.5.21-release-328-AS7717.8',
```

Comment:
Same for the kotlin, see plugin versions [here](https://plugins.jetbrains.com/plugin/6954-kotlin/versions/stable) , seems like we can use 203-1.5.21-release-316-IJ7717.8